python/mxnet/test_utils.py

```diff
@@ -1223,6 +1221,10 @@ def check_consistency(sym, ctx_list, scale=1.0, grad_req='write',
         Optional, When flag set to true,
         random input data generated follows uniform distribution,
         not normal distribution
+    rand_type: np.dtype
+        Optional, when input data is passed via arg_params,
```

Comment:
This piece of document seems a bit confusing, maybe you should also specify that this argument is for "controlling the data type of the randomly generated test data" etc. ?